developer/docs/help/guides/intro.md

```diff
@@ -13,7 +13,9 @@ There are two core applications included in Keyman Developer: the Keyman
 Developer IDE (formerly called TIKE), and the command line compiler kmc.
 
 [Watch a video](https://youtu.be/kwhgx_eX4Es) that highlights some of
-the Keyman Developer features.
+the Keyman Developer 14.0 features:
```

Comment:
I wonder if we just need one link for each of these videos. (And the video duration may be helpful)

```suggestion
Watch a video that highlights some of
the Keyman Developer 14.0 features (57:23)
```